client/my-sites/plans-grid/hooks/npm-ready/data-store/use-grid-plans.tsx

```diff
@@ -72,6 +73,8 @@ export type UsePricedAPIPlans = ( { planSlugs }: { planSlugs: PlanSlug[] } ) =>
 	[ planSlug: string ]: PricedAPIPlan | null | undefined;
 } | null;
 
+export type storageAddOnForPlan = AddOnMeta & { checkoutUrl: string };
```

Comment:
> why is this extended and defined here and not where AddOnMeta lives?

Let me know what you think though - whether we should keep add-on definitions centralised, or if it makes more sense to have this definition live closer to the plans data-store (where the use-grid-plans hook will live) 🤔

Reply:
I ended up removing these combined types in https://github.com/Automattic/wp-calypso/commit/5ca53a6f37850ac4389026ec1fa9d482f8d7d22b because we generate storage add-ons with the `checkoutLink` instead.

Left some thoughts about why in https://github.com/Automattic/wp-calypso/pull/83005#discussion_r1370992420